### ISSUE TITLE
dotnet ef tools 8.0.2 not compatible with .Net SDK 8.0.101?

### ISSUE THREAD

**Issue by m-gug** [dotnet/efcore#33092]
In our Azure DevOps pipeline we install the latest dotnet ef tools using:
`install --global dotnet-ef --version 8.*`
And after that we create a migration script with the tools.

Since the release of dotnet ef tools 8.0.2 yesterday, the creation of the migration script is failing with the this error:

> You must install or update .NET to run this application.
> 
> App: C:\Users\VssAdministrator\.dotnet\tools\dotnet-ef.exe
> Architecture: x64
> Framework: 'Microsoft.NETCore.App', version '8.0.2' (x64)
> .NET location: C:\Program Files\dotnet\
> 
> The following frameworks were found:
>   6.0.5 at [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
>   6.0.26 at [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
>   7.0.5 at [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
>   7.0.15 at [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
>   8.0.1 at [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
> 
> Learn more:
> https://aka.ms/dotnet/app-launch-failed
> 
> To install missing framework, download:
> https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=8.0.2&arch=x64&rid=win-x64&os=win10

For the pipeline we use MS hosted agents that have the .Net SDK 8.0.101 installed.

Is it intentional that the dotnet ef tools 8.0.2 is not compatible with the .Net SDK 8.0.101 or am I missing something.

EF Core version: 8
Target framework: NET 8.0

**Comment by wdhenrik** [comment]
Same issue here.  Unfortunate workaround is to force the unpatched version:
```
- task: DotNetCoreCLI@2
  displayName: Install EF tools
  inputs:
    command: custom
    custom:  'tool '
    arguments: install dotnet-ef --version 8.0.0
```
It would be nice if the Azure DevOps SDK versions were synchronized when a new update is released.

**Comment by m-gug** [comment]
@ajcvickers 
> Duplicate of #32782
Yes it seems to be the same problem, but the issue is closed but the problem still exists and i can't find a clear solution or statement in the linked issue.
Is really on purpose, that we have to pin the dotnet ef tools to a patch version - that sounds wrong.

**Comment by ajcvickers** [comment]
@m-gug `dotnet-ef` is a thin shim. There is no functional difference between 8.0.0, 8.0.1, and 8.0.2. Once problem is fixed in 8.0.3 and so won't require using a specific version again.

**Comment by m-gug** [comment]
@ajcvickers OK, I see, thanks for the clarification!
